### PR TITLE
pkp/pkp-lib#7592 Refactor reviewer-related emails

### PR DESCRIPTION
**Pull request by Vitaliy-1**

### CODE REVIEW

File: locale/en_US/manager.po [hunk 1]
```diff
@@ -1760,6 +1760,24 @@ msgstr "The currency of the payment amount, such as USD"
 msgid "emailTemplate.variable.site.siteTitle"
 msgstr "Name of the website when more than one journal is hosted"
 
+msgid "emailTemplate.variable.subscription.subscriberDetails"
+msgstr "Information about a subscriber, including user full name, affiliation, phone number and email"
```
Comment (NateWr) [inline locale/en_US/manager.po:1764]: > The subscriber's name, affiliation, phone number and email address

File: locale/en_US/manager.po [hunk 2]
```diff
@@ -1760,6 +1760,24 @@ msgstr "The currency of the payment amount, such as USD"
...
+
+msgid "emailTemplate.variable.subscription.subscriptionSignature"
+msgstr "Contact details of the subscription manager"
+
+msgid "emailTemplate.variable.subscription.subscriptionUrl"
+msgstr "The link to the subscription page"
```
Comment (NateWr) [inline locale/en_US/manager.po:1770]: > The URL to the subscription page

File: locale/en_US/manager.po [hunk 3]
```diff
@@ -1760,6 +1760,24 @@ msgstr "The currency of the payment amount, such as USD"
...
+
+msgid "emailTemplate.variable.subscription.expireDate"
+msgstr "The end date of the subscription"
```
Comment (NateWr) [inline locale/en_US/manager.po:1773]: > The date when the subscription ends

File: locale/en_US/manager.po [hunk 4]
```diff
@@ -1760,6 +1760,24 @@ msgstr "The currency of the payment amount, such as USD"
...
+
+msgid "emailTemplate.variable.subscription.subscriptionType"
+msgstr "subscription type summary, containing name, duration and cost"
```
Comment (NateWr) [inline locale/en_US/manager.po:1776]: A summary of the subscription's type, name, duration and cost

File: locale/en_US/manager.po [hunk 5]
```diff
@@ -1760,6 +1760,24 @@ msgstr "The currency of the payment amount, such as USD"
...
+
+msgid "emailTemplate.variable.subscription.membership"
+msgstr "subscription membership information"
```
Comment (NateWr) [inline locale/en_US/manager.po:1779]: Information about the subscriber's membership

**Review by NateWr — CHANGES_REQUESTED** [review]

File: locale/ar_IQ/emails.po [hunk 6]
```diff
@@ -347,30 +347,6 @@ msgstr ""
 msgid "emails.reviewResponseOverdueAutoOneclick.description"
 msgstr "هذه الرسالة يرسلها النظام آلياً عندما يتباطأ أحد المحكمين عن تقديم إجابة في الموعد المقرر (إنظر خيارات التحكيم ضمن: الإعدادات، المخطط الانسيابي، التحكيم) عندما تكون ميزة وصول المحكم بنقرة واحدة ممكنة. يجب أن تكون المهام المجدولة ممكنة ومعدة (إنظر ملف إعدادات الموقع)."
 
-msgid "emails.reviewRequestAttached.subject"
```
[the remark this answers is not in the document]
Comment (Vitaliy-1) [reply locale/ar_IQ/emails.po:350]: Yes, rebasing was a pain

File: classes/mail/variables/SubscriptionEmailVariable.inc.php [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+    protected static function description(): array
+    {
+        return
+        [
+            self::SUBSCRIBER_DETAILS => 'emailTemplate.variable.subscription.subscriberDetails',
+            self::SUBSCRIPTION_SIGNATURE => 'emailTemplate.variable.subscription.subscriptionSignature',
+            self::SUBSCRIPTION_URL => 'emailTemplate.variable.subscription.subscriptionUrl',
+            self::EXPIRE_DATE => 'emailTemplate.variable.subscription.expireDate',
+            self::SUBSCRIPTION_TYPE => 'emailTemplate.variable.subscription.subscriptionType',
+            self::MEMBERSHIP => 'emailTemplate.variable.subscription.membership',
```
[the remark this answers is not in the document]
Comment (Vitaliy-1) [reply classes/mail/variables/SubscriptionEmailVariable.inc.php:70]: oops

**Comment by Vitaliy-1** [comment]
@NateWr, it looks like the error during tests is due to `ui-library` not being synced with the current OJS main branch.

**Comment by NateWr** [comment]
Have you tried to pull the latest changes from the ui-library in and add them to your submodule commit? The text of the emaili template in the logs looks like none of the email variables are being changed, which suggests something maybe more significant is off here:

https://app.travis-ci.com/github/pkp/ojs/jobs/565390269#L2663

I don't think `{$submissionTitle}` or `{$signature}` would appear in the email composer without being rendered based on a recent commit. If you sync everything up and still run into a test failure, let me know and I'll look into it further.

**Pull request merged**